cookbook/examples/streamlit_apps/github_mcp_agent/agents.py

```diff
@@ -0,0 +1,55 @@
+import os
+from textwrap import dedent
+
+from agno.agent import Agent
+from agno.tools.mcp import MCPTools
+from agno.utils.log import logger
+from utils import get_model_from_id
+
+try:
+    from mcp import ClientSession, StdioServerParameters
+    from mcp.client.stdio import stdio_client
+except ImportError:
+    raise ImportError("MCP packages not installed. Please install with: pip install mcp")
+
+
+async def run_github_agent(message: str, model_id: str = "gpt-4o"):
+    if not os.getenv("GITHUB_TOKEN"):
+        return "Error: GitHub token not provided"
+
+    try:
+        server_params = StdioServerParameters(
```

Comment:
This is a pretty outdated form of connection. Why do you need to manually make the client session? This should be handled inside the MCPtools